### PR TITLE
added light-up 2

### PR DESCRIPTION
**Pull request by gadiex**
please review, @casey-collab

### CODE REVIEW

File: english/marshmallow.md [hunk 1]
```diff
@@ -94,4 +94,6 @@ Do you:
 
 [hide the lighter fork and rope in your pocket](hide/hide-chk.md)
 
-[make a necklace out of the marshmallows](necklace/necklace.md)
\ No newline at end of file
+[make a necklace out of the marshmallows](necklace/necklace.md)
+
+[light up the wall 2](light-up2/light-up2.md)
```
Comment (casey-collab) [inline english/marshmallow.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by gadiex** [comment]
please review, @casey-collab

File: english/marshmallow.md (hunk 1 above)
Comment (casey-collab) [inline english/marshmallow.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by gadiex** [comment]
please review, @casey-collab

File: english/marshmallow.md (hunk 1 above)
Comment (casey-collab) [inline english/marshmallow.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/marshmallow.md (hunk 1 above)
Comment (gadiex) [inline english/marshmallow.md]: broken link fixed. please review, @casey-collab
<https://github.com/casey-collab>

On Thu, Feb 5, 2015 at 4:59 PM, Casey Collaborator <notifications@github.com
> wrote:

> In english/marshmallow.md
> <https://github.com/udacity/create-your-own-adventure/pull/709#discussion_r24150243>
> :
>
> > \ No newline at end of file
> > +[make a necklace out of the marshmallows](necklace/necklace.md)
> > +
> > +[light up the wall 2](light-up2/light-up2.md)
>
> It looks like this link is broken. You can view the page on GitHub on your
> fork and click the link there to check if it is working correctly.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/udacity/create-your-own-adventure/pull/709/files#r24150243>
> .
>



-- 

Regards,



Cheng

**Comment by gadiex** [comment]
please review, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :fireworks:

**Pull request merged**